imports/ui/QuestionEditItem.jsx

```diff
@@ -12,6 +12,8 @@ import 'react-select/dist/react-select.css'
 import { Editor } from './Editor'
```

Comment:
when I go to create a new question, I get server side errors about "Cannot read property requireApprovedPublicQuestions" of undefined - so you probably need to wait in the render function of the question side bar before displaying, e.g. show loading until course is defined.